borgweb/static/index.js

```diff
@@ -0,0 +1,50 @@
+var logFilesList = []
+var logFilesListHTML = ""
+
+var success = function (data) {
+  logFiles = data.log_files
+}
+var parseAnchor = function () {
+  var url = window.location.href.toString()
+  var idx = url.indexOf("#")
+  var anchor = (idx != -1) ? url.substring(idx+1) : ""
+  if (anchor) {
+    var parts = anchor.split(';')
+    var partsParsed = {}
+    parts.forEach(function (e) {
+      var pair = e.split(':')
+      partsParsed[pair[0]] = pair[1]
+    })
+    return partsParsed
+  }
+}
+var updateLogFileList = function (logFiles) {
+  $.each(logFiles.log_files, function (key, value) {
+    logFilesListHTML += "<li><a href=\""
+      + "#log:" + value[0]
+      + "\""
+      + " onClick=\"displayThatLog("
+      + value[0] + ")\">"
+      + value[1] + "</a></li>"})
```

Comment:
this looks a bit ugly. can js do this in a prettier way?

like using different outer quotes so that escaping the inner ones is not required.
like using a template with placeholders instead of x + y + z?